sw/device/lib/base/testing/BUILD

```diff
@@ -0,0 +1,35 @@
+# Copyright lowRISC contributors.
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+
+package(default_visibility = ["//visibility:public"]) 
+
+cc_library(
+    name = "mock_mmio",
+    linkstatic = True,
```

Comment:
I don't think `linkstatic` is necessary here.  This cc_library is effectively testonly and will only get linked with cc_tests which run on the local x86 target rather than the riscv32 target.  Its perfectly fine for this to be subject to `cc_test`'s preferred linking scheme.

Reply:
Sounds good, thanks.